features/importing/work_import_errors.feature

```diff
@@ -4,13 +4,13 @@ Feature: Import Works
   As an author
   I want to create new works by importing them
 
-  @work_import_errors @wip
+  @work_import_errors
   Scenario: Entering a bogus URL
     Given basic tags
       And I am logged in as a random user
     When I go to the import page
       And I fill in "urls" with "http://bogus"
```

Comment:
Can we use the .invalid domain.

http://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml

6.4.  Domain Name Reservation Considerations for "invalid."

   The domain "invalid." and any names falling within ".invalid." are
   special in the ways listed below.  In the text below, the term
   "invalid" is used in quotes to signify such names, as opposed to
   names that may be invalid for other reasons (e.g., being too long).

   1.  Users are free to use "invalid" names as they would any other
       domain names.  Users MAY assume that queries for "invalid" names
       will always return NXDOMAIN responses.